format/index.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Store } from 'effector';
+
+export function format<T extends Array<Store<any>>>(
```

Comment:
Wouldn't it be better to allow in types not just stores, but primitives too?

Like this:
```ts
const $name = createStore('Bob').on(change, (name) => nameMap.get(name));
const $age = $name.map(name => ageMap.get(name));

const formatted = format`${window.BASE_STRING} ${$name} ${$age}`;
```
It works correctly without typescript:
https://share.effector.dev/EKLxy7A1
so why not support this at types level

Types for that can be [borrowed from `forest`'s types](https://github.com/effector/effector/blob/261007707c52ddd6995fc80e4042e6348a05c58c/src/forest/index.h.ts#L50) 

 Everything else looks cool, nice work